### DISCUSSION TITLE
Does autoLogin works along with Appium & using Typescript as the language?

### DISCUSSION THREAD

**Discussion by clundstedt-m** [codeceptjs/CodeceptJS#3866] [Q&A]
Hey there team!

We had success implementing the autoLogin plugin for the Web-based tests (using Playwright), but on the Mobile-end it seems like it "runs", but it actually doesn't saves anything or even gives a hint that it's actually missing something as it just passes the test but it doesn't actually runs the steps defined for the login itself.

Has anyone had success implementing the autoLogin plugin with Appium and TS (or JS).

Best regards!

**Comment by kobenguyent** [comment]
Hi, if I got it correctly, the autoLogin only works with web app, as it uses cookies to do the trick. 

https://github.com/codeceptjs/CodeceptJS/blob/3.x/lib/plugin/autoLogin.js
